docs/modules/integrations/pages/azure-devops-exporter.adoc

```diff
@@ -49,6 +51,35 @@ Please make sure that the `bdd.configuration.formats` property includes JSON val
 
 |===
 
+=== Test run properties
+
+[IMPORTANT]
+====
+* All test cases being exported should belong to the specified test plan.
+* Automated test runs can contain only automated test cases.
```

Comment:
we allow to have scenarios that contain only comment and consider them as manual, in case of XRay test run may contain both automated and manual tests, in case of AzureDevOps test run can contain either "manual" or automated

Reply:
I expect to have a similar description in docs with the corresponding links if necessary

Reply:
rephrase